packages/@ionic/cli-utils/src/lib/project/angular/utils.ts

```diff
@@ -24,19 +24,19 @@ export async function readAngularConfigFile(p: string): Promise<AngularConfig> {
   return angularJson;
 }
 
-export async function addCordovaEngineForAngular(project: IProject, platform: string): Promise<void> {
+export async function addCordovaEngineForAngular(project: IProject, platform: string, appName = 'app'): Promise<void> {
   debug('Adding Cordova engine for platform: %s', platform);
-  const srcDir = await project.getSourceDir();
   const platformWWW = path.resolve(project.directory, 'platforms', platform, 'platform_www');
   const angularJsonPath = path.resolve(project.directory, ANGULAR_CONFIG_FILE);
   const angularJson = await readAngularConfigFile(angularJsonPath);
-  const app = angularJson.projects.app;
+  const angularApp = angularJson.projects[angularJson.defaultProject || appName];
```

Comment:
Shouldn't we use `appName` if provided, otherwise use `angularJson.defaultProject`? This seems like it wouldn't work if a `defaultProject` is defined but `--project=something` is used.